iree_tests/benchmarks/benchmark_sdxl_rocm.py

```diff
@@ -0,0 +1,63 @@
+import os
+from collections import namedtuple
+
+benchmark_dir = os.path.dirname(os.path.realpath(__file__))
+
+def launchCommand(scriptcommand):
+    print("Launching:", scriptcommand, "[ Proc:", os.getpid(), "]")
+    try:
+        ret = os.system(scriptcommand)
+        return ret
+    except OSError as errormsg:
+        print(
+            "Invoking ",
+            scriptcommand,
+            " failed:",
+            errormsg,
+        )
+        return 1
+
+def run_sdxl_rocm_benchmark():
+    scriptcommand = (
+        "bash "
+        + f"{benchmark_dir}/benchmark_sdxl_rocm.sh"
+    )
+    return launchCommand(scriptcommand)
```

Comment:
It's a bit awkward to have a python script call a bash script when the python script could run all steps itself. See how https://github.com/iree-org/iree/blob/main/experimental/regression_suite/ireers/fixtures.py has a fixture for benchmarking

Windows also doesn't have the `bash` command by default, so this would not be portable to all systems in the way that a python script would.

Reply:
Switched to pure python implementation